osconfig_tests/test_suites/guestpolicies/testfiles/exec_test

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
i don't quite follow this paradigm. this will get uploaded so it can be used in an exec step test? where do exec_test_windows.* files get uploaded? also, i don't think we should be storing any binary artifacts like tarballs in git

Reply:
Currently it uploads the recipes to a temporary bucket it creates. Generally I don't store these things in source control but these shouldn't change often, would it be better to store them in an open bucket instead?

Reply:
you mean it uploads these files as artifacts for use in a recipe? i'm fine with that, in general i also would not store in git. for such simple files i am just going to say it's fine for now, though.